bin/oref0-pump-loop.sh

```diff
@@ -917,6 +920,14 @@ function compare_with_fullhistory() {
   fi
 }
 
+function update_glucose_noise() {
+    if check_pref_bool .calc_glucose_noise false; then
```

Comment:
This seems backwards?

Reply:
It seems to work right.  It defaults to false (the second argument to check_pref_bool) if .calc_glucose_noise is not set, so the if expression only evaluates to true if .calc_glucose_noise is in the preferences.json file and is set to true.

Reply:
Ok, that's a really confusing implementation of check_pref_bool (not your fault). I read it to be equivalent to "if calc_glucose_noise === false".

Reply:
Yes! that's the way I wrote it first, but it didn't work! 😄